multinet/auth/google.py

```diff
@@ -139,9 +148,18 @@ def authorized(state: str, code: str) -> ResponseWrapper:
         user.save()
 
     return_url = session.pop("return_url", default_return_url())
-    resp: ResponseWrapper = make_response(redirect(ensure_external_url(return_url)))
+    resp: Response = make_response(redirect(ensure_external_url(return_url)))
 
     encoded_login_token = encode_auth_token(create_login_token(user.get_session()))
-    resp.set_cookie(MULTINET_LOGIN_TOKEN, encoded_login_token)
+
+    # If running in development, don't set secure and samesite cookie attributes
+    development = current_app.config.get("ENV") == "development"
+    resp.set_cookie(
+        MULTINET_LOGIN_TOKEN,
+        value=encoded_login_token,
+        secure=True if not development else False,
+        samesite="None" if not development else None,
```

Comment:
From MDN, "None" means:
> Cookies will be sent in all contexts, i.e sending cross-origin is allowed.

Is this the behavior we want? I guess so to interface with applications that are not on multinet.app

Reply:
Yes, a good example is preview deploys. We also do have CORS set up, so a request from another origin which is not in our list will be blocked by the browser.